game/atmosphere.rpy

```diff
@@ -1,8 +1,24 @@
-# Sky types for the classroom
-image sky_day overcast = "mod_assets/backgrounds/classroom/sky_day_overcast.png"
-image sky_day rain = "mod_assets/backgrounds/classroom/sky_day_rain.png"
-image sky_day sunny = "mod_assets/backgrounds/classroom/sky_day_sunny.png"
-image sky_day thunder = "mod_assets/backgrounds/classroom/sky_day_thunder.png"
+# Weather data
+default persistent.jn_weather_api_key = None
+default persistent.jn_weather_api_configured = False
+
+# Location data
+default persistent.jn_player_latitude = None
+default persistent.jn_player_longitude = None
```

Comment:
Since we'd always be using these together, it might be best to store a tuple `persistent._jn_player_latlon` or something